modules/gitops-release-notes-1-11-0.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-11-0_{context}"]
+= Release notes for {gitops-title} 1.11.0
+
+{gitops-title} 1.11.0 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="new-features-1-11-0_{context}"]
+== New features
+
+The current release adds the following improvement:
+
+* With this update, you can selectively disable the `redis` and `application-controller` components for an Argo CD instance in a specified namespace. These components are enabled by default.
+To disable a component, set the `enabled` flag to `false` in the `.spec.<component>.enabled` field of the Argo CD Custom Resource (CR). link:https://issues.redhat.com/browse/GITOPS-3723[GITOPS-3723]
++
+.Example: Disabling the `redis` and `application-controller` components
+[source, yaml]
```

Comment:
Per OpenShift documentation [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output:~:text=do%20not%20insert%20a%20space):
> do not insert a space in between the [source] tag and the metadata.

```suggestion
[source,yaml]
```

Reply:
ACK.
Addressed